implementations/rust/ockam/ockam_vault/Cargo.toml

```diff
@@ -33,7 +33,11 @@ sha2 = "0.9"
 x25519-dalek = "1.1"
 zeroize = { version = "1.1", features = ["zeroize_derive"] }
 tracing = "0.1.26"
+either = "1.6.1"
```

Comment:
Is it used?

Reply:
No, it is leftover from my experiments. Removed.